### PR TITLE
Add rel="noopener" to all external (_blank) links

### PR DESCRIPTION
**Pull request by atomfrede**
We are missing `rel="noopener"` for all external links, I think we should have that for security reasons.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -414,13 +414,13 @@ <h5 class="text-left"><i class="fa fa-database fa-fw"></i> <a class="link" href=
                                     <div class="footer-col col-md-4">
                                         <ul class="list-inline">
                                             <li>
-                                                <a href="https://twitter.com/java_hipster" target="_blank" class="btn-social btn-outline"><i class="fa fa-fw fa-twitter"></i></a>
+                                                <a href="https://twitter.com/hipster" target="_blank" rel="noopener" class="btn-social btn-outline"><i class="fa fa-fw fa-twitter"></i></a>
```
Comment (deepu105) [inline _layouts/default.html]: This seems like a typo

Comment (atomfrede) [reply _layouts/default.html]: Yes :man_facepalming:

**Comment by deepu105** [comment]
Hey could you look at the diff on the PR please. I'm seeing merge conflict statements on HTML

**Comment by atomfrede** [comment]
> Hey could you look at the diff on the PR please. I'm seeing merge conflict statements on HTML

Damn, what did code do there !?

**Comment by deepu105** [comment]
You can enable autosave in Code too. I never trust myself so I always look at the diff of my PRs and mostly catch stuff like this :wink:

**Pull request merged**